test/unit/services/test_catalogue_category.py

```diff
@@ -664,9 +698,6 @@ def test_update_change_catalogue_item_properties_when_has_children(
     )
     # pylint: enable=duplicate-code
 
-    # Mock so child elements found
-    catalogue_category_repository_mock.has_child_elements.return_value = True
-
```

Comment:
Why was this removed? It looks like it should be here based on the name, but then again the tests are passing.

Reply:
to be honest I can't remember why, it probably was on accident, but the test passes with and without it so up to you if I should add it back in

Reply:
I think it should be as the others typically have them - my guess is that the count_documents function is mocked and so returns an MagicMock or something that is being evaluated to true, which works but I think its clearer if its there.